impress-org/givewp
# e2e testing is now scaffolded (via Cypress and wp-env)

Resolves #5511 

## Description
This PR scaffolds new e2e tests powered by Cypress and wp-env, and removes deprecated e2e tests (which were run with Jest and Puppeteer). After running `npm install` and ensuring they have Docker Desktop (or equivalent Docker dependencies) installed, developers can now run the `test:e2e` command to begin local e2e testing with Cypress.

## Affects
This PR affects existing e2e testing, and introduces the `npm run test:e2e` command to run new tests with Cypress.

## Visuals
https://www.loom.com/share/182a0f6fe44245e7b436697fdd9fe032

## Pre-review Checklist

## Testing Instructions
1. Install Docker Desktop (or equivalent dependencies)
2. Run `npm install` to ensure Cypress is installed
3. Run `npm run test:e2e` to open Cypress
4. Select the `sample_spec.js test`, check that it loads the Reports dashboard as expected
